enterprise/server/remote_execution/containers/ociruntime/ociruntime.go

```diff
@@ -55,7 +55,7 @@ var (
 	cpuSharesEnabled = flag.Bool("executor.oci.cpu_shares_enabled", false, "Enable CPU weighting based on task size.")
 	dns              = flag.String("executor.oci.dns", "8.8.8.8", "Specifies a custom DNS server for use inside OCI containers. If set to the empty string, mount /etc/resolv.conf from the host.")
 	netPoolSize      = flag.Int("executor.oci.network_pool_size", -1, "Limit on the number of networks to be reused between containers. Setting to 0 disables pooling. Setting to -1 uses the recommended default.")
-	fakeCPUInfo      = flag.Bool("executor.oci.fake_cpu_info", false, "Use lxcfs to fake cpu info inside containers.")
+	enableLxcfs      = flag.Bool("executor.oci.enable_lxcfs", false, "Use lxcfs to fake cpu info inside containers.")
```

Comment:
nit: to match `cpu_shares_enabled` above

```suggestion
	enableLxcfs      = flag.Bool("executor.oci.lxcfs_enabled", false, "Use lxcfs to fake cpu info inside containers.")
```